guides/choosing_mono_or_multi_repo.md

```diff
@@ -0,0 +1,96 @@
+However mono repos may also tightly coupled collaboration needs across independent teams.  This can lead to conflicts and bottlenecks in the development process.
+
+Multi repos are more difficult to collaborate as data is distributed across multiple repositories.  This can mean key data is missed which leads to risk.
+
+### CI/CD
+
+Mono repos simplify CI/CD as multiple projects can be built and tested together. This can reduce the complexity of creating and managing multiple pipelines and dependencies.
+
+However, mono repos can also lead to longer build times and increased complexity in managing dependencies, particularly if multiple technology choices are used.
+
+Additional tooling and complexity may be required to independently build and test projects within a mono repo depending on the scale.
+
+Multi repos require more configuration for CI/CD as each project needs its own pipeline which can lead to duplication of effort and complexity in co-ordination of pipeline updates.
+
+The advantage is a more optimised pipeline for each project.
+
+### Release process
+
+Mono repos are best suited for projects that require all code to be released together.
+
+Multi repos are best suited for projects that require independent deployments.
+
+The scale of the service should again be considered as releasing or rolling back a large scale mono repo code base can add unnecessary risk and complexity.
+
+### Commit history
+
+Mono repos have a single commit history which can make it easier to track changes and understand the history of the codebase.
+
+Multi repos have multiple commit histories which can make it more difficult to track changes and understand the history of the codebase, however the history is more focused.
+
+### Versioning
+
+For simple projects a single version may be sufficient within a mono repo, however for more complex projects a versioning strategy should be considered.  This may involve additional tooling to independently version and release projects.
+
+Multi repos can have independent versioning strategies for each project which can simplify the release process.
+
+### Security
+
+Mono repos can simplify security as access control and visibility is centralised.  Dependency scanning tools are also simpler to maintain.
+
+Mono repos can also increase the risk of a single point of failure and increase the blast radius of a security incident.
+
+It may also be difficult to restrict access to sensitive code or data within a mono repo.
+
+Multi repos can increase the complexity of security as access control and visibility is distributed across multiple repositories increasing the risk stale users may have access to sensitive data.
+
+Multi repos allow for more granular access control.
+
+### Ownership
+
+Mono repos are best suited when a single team owns all the code and is expected to maintain it.
+
+Multi repos are best suited when multiple teams own different parts of the codebase to avoid tight coupling different delivery strategies.
```

Comment:
We could probably say some stronger than "best suited" here?

Reply:
Generally changed the tone to be more opinionated, but specifically here too.